Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -31,8 +31,14 @@ function OnUpdate_GasGenerator( furniture, deltaTime )
         end
     end
 	return
+	furniture.SetAnimationState("running")
 end
 
+function OnPowerOff_GasGenerator( furniture, deltaTime )
```

Comment:
We have an issue #765, where it looks like we agree on `funitureName_functionName`. But the furniture names might change in the future.